config/roles.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    "roles" => [
+        ["name" => "super_admin"],
+        ["name" => "admin"],
+        ["name" => "user"],
```

Comment:
```suggestion
        "super_admin",
        "admin",
        "user",
```

that should be simple :D